.github/workflows/spelling-all.yaml

```
@@ -0,0 +1,30 @@
+name: Check Spelling - All
+
+on:
+  push:
+    branches:
+      - main
+      - RPSaaSMaster
+      - typespec-next
+
+  schedule:
+    # Run every day at 12:06 UTC
+    - cron: "6 12 * * *"
+  workflow_dispatch:
+
+jobs:
+  check-spelling:
```

```suggestion
  spellcheck:
    name: SpellCheck
```

Our convention for single-job workflows, is to use the same name as the workflow itself.